disconnected/updating/index.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-disconnected-updates"]
+= About cluster updates in a disconnected environment
+include::_attributes/common-attributes.adoc[]
+:context: about-disconnected-updates
+
+toc::[]
+
+A disconnected environment is one in which your cluster nodes cannot access the internet, or where you want to manage update recommendations and release images locally for policy or performance purposes.
+This section covers mirroring {product-title} images, managing an OpenShift Update Service, and performing cluster updates in a disconnected environment.
+
+[id="about-disconnected-updates-mirroring_{context}"]
+== Mirroring {product-title} images
+To update your cluster in a disconnected environment, your cluster environment must have access to a mirror registry that has the necessary images and resources for your targeted update.
+A single container image registry is sufficient to host mirrored images for several clusters in the disconnected network.
+The following page has instructions for mirroring images onto a repository in your disconnected cluster:
+
+* xref:../../disconnected/updating/mirroring-image-repository.adoc#mirroring-ocp-image-repository[Mirroring {product-title} images]
+
+[id="about-disconnected-updates-update_{context}"]
+== Performing a cluster update in a disconnected environment
+
+You can use one of the following procedures to update a disconnected {product-title} cluster:
+
+* xref:../../disconnected/updating/disconnected-update-osus.adoc#updating-disconnected-cluster-OSUS[Updating a cluster in a disconnected environment using the OpenShift Update Service]
```

Comment:
```suggestion
* xref:../../disconnected/updating/disconnected-update-osus.adoc#updating-disconnected-cluster-OSUS[Updating a cluster in a disconnected environment by using the OpenShift Update Service]
```

Reply:
Although I agree that maybe there's a preposition missing in this title, the actual title doesn't have "by" in it. Thanks for catching this anyways